src/extract/cod.py

```diff
@@ -18,3 +19,36 @@ def get_adm0(raw_data_dir: str, output_filename: str):
                              [HDX_YEMEN_FILENAME])[HDX_YEMEN_FILENAME]
     shutil.move(os.path.join(raw_data_dir, filepath_hdx),
                 output_filename)
+
+# HDX COD Admin 1
+def get_adm1_snakemake():
```

Comment:
These methods are all exactly the same as for admin 0. If you still want to have different snakemake rules for each admin level, I would just define a single COD extraction entrypoint, and call the same method for each rule. (You could just a single rule and call it `extract_cod_adm0_to_adm3` or something like that)